d20potz.py

```diff
@@ -169,7 +169,47 @@ async def send_cards(
     if media_list:
         await context.bot.send_media_group(chat_id=chat_id, media=media_list)
 
+async def check_player_name(context: ContextTypes.DEFAULT_TYPE, chat_id, player_name):
+    player_list = CONFIG.order.lower().split()
+    if player_name not in player_list:
+        await context.bot.send_message(
+            chat_id=chat_id,
+            text="{} is not one of {}".format(
+                spell_hero_name(player_name), player_list
+            ),
+        )
+        return False
+    return True
+
+def set_claim_status(chat_id, player_name, user_id, claim: bool):
+    claim_key = f"player_claim_{chat_id}_{player_name}".encode("utf-8")
+    logging.info(f"Writing claim status for key {claim_key}")
+    if claim:
+        DB.Put(claim_key, str(user_id).encode("utf-8"))
+    else:
+        DB.Delete(claim_key)
 
+def get_player_by_user(chat_id, user_id):
+    player_list = CONFIG.order.lower().split()
+    
+    for player_name in player_list:
+        claim_key = f"player_claim_{chat_id}_{player_name}".encode("utf-8")
+        claimant = DB.Get(claim_key, default = None)
+        if claimant != None and claimant.decode() == str(user_id):
```

Comment:
```suggestion
        if claimant is not None and claimant.decode() == str(user_id):
```

Reply:
Kinda similar to java equals(). A class can override `== None`, but not `is None`